src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/CommentsIndentationCheck.java

```diff
@@ -34,7 +34,7 @@
  * Comments are indented at the same level as the surrounding code.
  * Detailed info about such convention can be found
  * <a href=
- * "https://checkstyle.org/styleguides/google-java-style-20170228.html#s4.8.6.1-block-comment-style">
+ * "https://checkstyle.org/styleguides/google-java-style-20180523/google.github.io/styleguide/javaguide.html#s4.8.6.1-block-comment-style">
```

Comment:
I am not seeing `google-java-style-20180523.html`.
I assume `javaguide` is the `google-java-style-20180523`? What is the purpose of putting in such a weird nested folder instead of where it was before?

Reply:
@Vampire , please rebase on latest and explain extra folders.
I think you simply copied them from github repository (previously I did "page save" in browser, as it was simpler at that time, I do not remember why), so it is ok, we just need to know.

Reply:
> I am not seeing `google-java-style-20180523.html`.
> I assume `javaguide` is the `google-java-style-20180523`?

Exactly

> What is the purpose of putting in such a weird nested folder instead of where it was before?

Before @romani just used the browsers "Save page as" functionality, which most often does not work as intended and for example added these ugly effects I highlighted in the issue, as the page-save saves stuff that was added dynamically via JavaScript. But the JavaScript is also still there, so things are doubled. And the dynamically added images are not where expected, so you get those missing images displayed.

I used the software "WinHTTrack Website Copier" to do a proper site download that displays correctly, including all resources where they are expected.

Reply:
> I used the software "WinHTTrack Website Copier"

After some time of thinking on this ...
lets not depend on some tool here, I do not know such tool, and I will not be able to use it as it is  for Windows.
Please update please folders to be at `google-java-style-20180523` I hope style guide html style will not be changed in recent few years. So all we will need to update periodically is html file. So some any tool, and even "Save as" will work.

Reply:
> lets not depend on some tool here, I do not know such tool, and I will not be able to use it as it is for Windows.

It is not.
Don't get trapped by the "Win" in the name, it is just the Windows build of HTTrack which is available for Windows, Linux, OSX and even Android: http://www.httrack.com/page/2/

> Please update please folders to be at `google-java-style-20180523` I hope style guide html style will not be changed in recent few years. So all we will need to update periodically is html file. So some any tool, and even "Save as" will work.

The nice thing with HTTrack is, that it properly downloads websites and all related things that are necessary for offline or cached version.
Actually in this case it is not necessary yes, because it the actual HTML that is stored in the Google repository, so we can just copy in the files.
But keep it in mind for probably future supported style guides where the source is not the actual HTML, there HTTrack or any other proper website copier is much better and more reliable than using "Save as" in the browser which will usually save JS-post-processed versions of the pages which end up unnice like it was with the Google Styleguide here too.